source/voice_control/s3_box_voice_assistant.markdown

```diff
@@ -46,7 +45,18 @@ Before you can use this device with Home Assistant, you need to install a bit of
 
    - Your ESP32-S3-BOX is connected to Home Assistant over Wi-Fi. You can now move it to any place in your home with a USB power supply.
 
-10. Congratulations! You can now voice control Home Assistant via a ESP32 device with a display. Now give some commands.
+10. If you want, you can process the wake word on the ESP32-S3 device, rather than on your Home Assistant server. (The server is the device where Home Assistant is installed, for example on Home Assistant Green).
+    - Under **Devices**, on the ESP32-S3-BOX entry, select **Device** to open the device page.
+    - Under **Wake word engine location**, select **On device**, if you want your wake word to be processed locally.
```

Comment:
```suggestion
    - Under **Wake word engine location**, select **On device**, if you want your wake word to be processed on the device itself, and not in Home Assistant.
```

Note; We avoid calling this "flavor" local wake word. Both are local in regards to our definition (NEither use cloud)